src/js/apps/globals/nav_app.js

```diff
@@ -13,7 +13,7 @@ const i18n = intl.globals.nav;
 const appNavMenu = new Backbone.Collection([
   {
     onSelect() {
-      Radio.trigger('event-router', 'patients:all');
+      Radio.trigger('event-router', 'worklist', 'owned-by-me');
```

Comment:
tried going to `default`...but that didn't work, so I did this instead.

Reply:
Why because the test failed or because it didn't go to the owned-by-me list?

Reply:
it didn't go to the owned-by-me list